PrebidMobile/PrebidServerAdapter/src/main/java/org/prebid/mobile/prebidserver/PrebidServerAdapter.java

```diff
@@ -139,8 +139,7 @@ public void onServerResponded(JSONObject response) {
                                         if (targetingKeywords != null && targetingKeywords.length() > 0) {
                                             BidResponse newBid = null;
                                             if (Prebid.useLocalCache()) {
-                                                String format = targetingKeywords.getString("hb_creative_loadtype");
-                                                String cacheId = CacheManager.getCacheManager().saveCache(bid.toString(), format);
+                                                String cacheId = CacheManager.getCacheManager().saveCache(bid.toString(), "html");
```

Comment:
Should we change the CacheManager to remove this conditional check for format?

We are now only saving on web

Reply:
We're upgrading to 1.0, let's leave it like this,